nav.go

```diff
@@ -525,53 +548,46 @@ func (nav *nav) preview(path string, win *win) {
 	}
 }
 
-func (nav *nav) previewClear() {
-	if len(gOpts.cleaner) != 0 && nav.volatilePreview {
-		nav.volatilePreview = false
+func (nav *nav) preview(path string) {
+	nav.previewChan <- path
+}
 
-		cmd := exec.Command(gOpts.cleaner)
-		if err := cmd.Run(); err != nil {
-			log.Printf("cleaning preview: %s", err)
-		}
-	}
+func (nav *nav) previewClear() {
```

Comment:
Same as above, putting it inline can make it easier to follow I think.

Reply:
OK.